### ISSUE TITLE
False positive on empty select block

### ISSUE THREAD

**Issue by nobu-k** [mgechev/revive#698]
**Describe the bug**

`empty-block` complains about `select {}` that `empty-block: this block is empty, you can remove it`.

**To Reproduce**

```
select {}
```

**Expected behavior**

It shouldn't be an error as it's a valid way to block forever.

**Logs**

```
empty-block: this block is empty, you can remove it (revive)
```

**Desktop (please complete the following information):**
 - OS: Ubuntu 20.04
 - Version of Go: 1.18.1

**Additional context**

I used revive via golangci-lint 1.45.2.

**Comment by chavacava** [comment]
Hi @nobu-k, thanks for creating the issue.
IMO, the frequency of use of an empty `select` is very low to deserve a refactoring of the rule.
For these rare cases I suggest to [deactivate the linter rule at the specific line](https://github.com/mgechev/revive#comment-directives): `//revive:disable-line:empty-block`

**Comment by nobu-k** [comment]
It's definitely rare and I didn't even see it for a long time, so I totally agree with your opinion.

I've already applied the workaround you mentioned (via golangci-lint's `// nolint`), but I just wanted to let you know about the issue in case it was easy to fix. I'm closing this issue. Thanks for the clarification!
